### ISSUE TITLE
bug: Ionic Select not Selectable with Screen Reader on

### ISSUE THREAD

**Issue by brandiwerner** [ionic-team/ionic-framework#21805]
# Bug Report

**Ionic version:**

[ ] **4.x**
[x] **5.x**

**Current behavior:**

With a phone's screen reader on, drag finger over elements in app. When element is over the [Selector ](https://ionicframework.com/docs/api/select) element the screen reader will not read it out or focus on it. The only way to focus on the selector is to use the swiping gesture to swipe to and from elements. When using the swiping method you have to swipe over the title and place holder text before the screen reader get to and reads the selector as a button.

**Expected behavior:**

With a phone's screen reader on, drag finger over the selector element. The selector element should be read out as one whole element. When swiping from element to element the selector element should be read as one element and not reading the ionTitle and placeholder text being read separately from the button itself.

**Steps to reproduce:**

- Create an Ionic app
- Create an IonSelect
- Deploy app as a pwa and test on ios or android phone
- Drag finger over the selector (drop down) and notice it is not read out
- Swipe to the selector element and notice each piece in the ion item is read out instead of being read out as a whole

**Related code:**

**Other information:**

**Ionic info:**

I am coding in Ionic React.

**Comment by liamdebeasi** [comment]
Thanks for the issue! This issue is being closed due to the lack of a code reproduction. If this is still an issue with the latest version of Ionic, please create a new issue and ensure the template is fully filled out.

Thank you for using Ionic!
